internal/proto/authd/authd.proto

```diff
@@ -166,6 +168,24 @@ message GetGroupByIDRequest{
   uint32 id = 1;
 }
 
+message SetUserIDRequest {
+  string name = 1;
+  uint32 id = 2;
+}
+
+message SetUserIDResponse {
+  repeated string warnings = 1;
+}
+
+message SetGroupIDRequest {
+  string name = 1;
+  uint32 id = 2;
+}
+
+message SetGroupIDResponse {
+  repeated string warnings = 1;
+}
```

Comment:
Given we're likely also adding support for setting the user shell and probably GECOs and maybe in future (with user DB metadata) more fields, isn't better to just have a single `SetUserProperties` message when we handle it all?

As in the daemon side we'll have a single lock place instead of having to manage it for each case we will handle.

Reply:
> As in the daemon side we'll have a single lock place

what do you mean with lock place?

I currently don't see the advantage of a single `SetUserProperties` message. The good thing is that we don't have to be very careful about this API between authd and authctl, because we ship them both in the same package, so we can always change it however we want.